src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Operation/NewAzureVMCommand.cs

```diff
@@ -300,6 +302,17 @@ public override void ExecuteCmdlet()
             }
             else
             {
+                var resourceClient =
```

Comment:
Please add an entry in the changelog for this change

Reply:
OK